### PR TITLE
fix import documentation for string separated values

### PR DESCRIPTION
**Pull request by ScottSuarez**
fix import documentation for string separated values.
b/368130892#comment14

closes https://github.com/hashicorp/terraform-provider-google/issues/19453#issuecomment-2438737436

**Release Note Template for Downstream PRs (will be copied)**

See [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) for guidance.

```release-note:none

```

### CODE REVIEW

File: mmv1/products/securitycenter/NotificationConfig.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ self_link: '{{name}}'
 create_url: 'organizations/{{organization}}/notificationConfigs?configId={{config_id}}'
 update_verb: 'PATCH'
 update_mask: true
+import_format:
```
Comment (melinath) [inline mmv1/products/securitycenter/NotificationConfig.yaml:34]: Why is this one necessary?

File: mmv1/products/securitycenter/ProjectNotificationConfig.yaml [hunk 2]
```diff
@@ -32,6 +32,8 @@ self_link: '{{name}}'
 create_url: 'projects/{{project}}/notificationConfigs?configId={{config_id}}'
 update_verb: 'PATCH'
 update_mask: true
+import_format:
```
Comment (melinath) [inline mmv1/products/securitycenter/ProjectNotificationConfig.yaml:35]: Same question

File: mmv1/products/securitycenterv2/OrganizationNotificationConfig.yaml [hunk 3]
```diff
@@ -32,6 +32,9 @@ self_link: '{{name}}'
 create_url: 'organizations/{{organization}}/locations/{{location}}/notificationConfigs?configId={{config_id}}'
 update_verb: 'PATCH'
 update_mask: true
+import_format:
+  - '{{project}} {{name}}'
```
Comment (melinath) [inline mmv1/products/securitycenterv2/OrganizationNotificationConfig.yaml:36]: why does this one need `{{project}}` when the other two similar ones didn't?

File: tpgtools/main.go [hunk 4]
```diff
@@ -562,6 +562,7 @@ func generateProductsFile(fileName string, products []*ProductMetadata) {
 
 var TemplateFunctions = template.FuncMap{
 	"title":                           strings.Title,
+	"containz":                        strings.Contains,
```
Comment (melinath) [inline tpgtools/main.go:565]: I'm a little unclear why this seems to be working as-is?

```suggestion
	"contains":                        strings.Contains,
```

**Review by melinath — COMMENTED** [review]
This seems to add a bunch of import formats that are like `{{project}}/{{name}}`, which I wasn't expecting - just want to double-check that those also seem to work?

File: tpgtools/main.go (hunk 4 above)
Comment (ScottSuarez) [reply tpgtools/main.go:565]: loool yeah this is not needed

File: mmv1/products/securitycenterv2/OrganizationNotificationConfig.yaml (hunk 3 above)
Comment (ScottSuarez) [reply mmv1/products/securitycenterv2/OrganizationNotificationConfig.yaml]: This one is wrong

**Review by melinath — APPROVED** [review]
This seems to add a bunch of import formats that are like `{{project}}/{{name}}`, which I wasn't expecting - just want to double-check that those also seem to work?

**Pull request merged**